src/Microsoft.DotNet.Interactive.Document/Microsoft.DotNet.Interactive.Document.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
+    <PackageId Condition="'$(PackageId)'==''">Microsoft.DotNet.Interactive.Document</PackageId>
```

Comment:
"Documents" would be more conventional for a namespace.

Reply:
done